devito/arch/compiler.py

```diff
@@ -614,6 +614,8 @@ def __init_finalize__(self, **kwargs):
         elif isinstance(platform, Cpu64):
             if language == 'openmp':
                 self.cflags.append('-mp')
+            if isinstance(platform, NvidiaArm):
+                self.cflags.append(f'-mcpu={platform.march}')
```

Comment:
I guess you cross-checked that this is not double-printed in the compilation line right? Otherwise OK